leakcanary-android-core/src/main/java/leakcanary/internal/HeapDumpTrigger.kt

```diff
@@ -403,20 +378,9 @@ internal class HeapDumpTrigger(
     notificationManager.notify(R.id.leak_canary_notification_retained_objects, notification)
```

Comment:
There are still 2 notify calls in this class, did you mean to keep those in place?

Reply:
No. I did not move everything yet, as I wanted to ensure that this approach looks reasonable as per: https://github.com/square/leakcanary/pull/2395/files/8efb01b46c6ca41eda8585b15612b64a7f9f5e04#r909122228

Reply:
I see. I was wondering if some notifications might prove trickier to move. I'd definitely want to do it all, that's for sure. Let's start there and see if there's any blocker.